### PR TITLE
Regular Sequence changes

### PR DESCRIPTION
**Pull request by lilyclements**

### CODE REVIEW

File: instat/dlgRegularSequence.vb [hunk 1]
```diff
@@ -44,43 +44,45 @@ Public Class dlgRegularSequence
...
+
         ucrInputInStepsOf.SetParameter(New RParameter("by", 2))
         ucrInputInStepsOf.AddQuotesIfUnrecognised = False
         ucrInputInStepsOf.SetValidationTypeAsNumeric(dcmMin:=0)
+
         ucrDataFrameLengthForRegularSequence.SetDataFrameSelector(ucrSelectDataFrameRegularSequence)
 
+        ucrPnlSequenceType.AddRadioButton(rdoNumeric)
+        ucrPnlSequenceType.AddRadioButton(rdoDates)
+        ucrPnlSequenceType.bAllowNonConditionValues = True
```
Comment (lilyclements) [inline instat/dlgRegularSequence.vb]: This boolean should not be applied to a ucrPnl

File: instat/dlgRegularSequence.vb [hunk 2]
```diff
@@ -94,19 +96,20 @@ Public Class dlgRegularSequence
         'ucrInputFrom.Visible = False
         ' ucrInputTo.Visible = False
         CheckSequenceLength()
-    End Sub
 
-    Private Sub ucrChkDefineAsFactor_Load(sender As Object, e As EventArgs) Handles ucrChkDefineAsFactor.Load
```
Comment (lilyclements) [inline instat/dlgRegularSequence.vb]: I removed this from it's own sub

File: instat/dlgRegularSequence.vb [hunk 3]
```diff
@@ -94,19 +96,20 @@ Public Class dlgRegularSequence
...
+        rdoDates.Enabled = False
     End Sub
 
     Private Sub SetDefaults()
         clsRepFunction = New RFunction
         clsSeqFunction = New RFunction
         ucrSelectDataFrameRegularSequence.Reset()
         ucrNewColumnName.Reset()
+
         rdoNumeric.Checked = True
```
Comment (lilyclements) [inline instat/dlgRegularSequence.vb]: This should not be checked in this way. Have a look at another dialog with radio buttons controlling the base function (e.g. Transform Text, Delete Rows or Columns) to see how to do it

File: instat/dlgRegularSequence.vb (hunk 1 above)
Comment (AlexSananka) [inline instat/dlgRegularSequence.vb]: This was fixing a developer error,I have tried figure out how to solve this but I couldn't do you have any suggestions to solve this?

File: instat/dlgRegularSequence.vb (hunk 2 above)
Comment (AlexSananka) [inline instat/dlgRegularSequence.vb]: OK

File: instat/dlgRegularSequence.vb (hunk 1 above)
Comment (lilyclements) [inline instat/dlgRegularSequence.vb]: Lets discuss on your branch so others can input if they wish

File: instat/dlgRegularSequence.vb (hunk 3 above)
Comment (AlexSananka) [inline instat/dlgRegularSequence.vb]: I have manage to change this.It is fixed now

**Pull request merged**